src/NATS.Client.JetStream/Models/ClusterInfo.cs

```diff
@@ -22,4 +29,40 @@ public record ClusterInfo
     [System.Text.Json.Serialization.JsonPropertyName("replicas")]
     [System.Text.Json.Serialization.JsonIgnore(Condition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingDefault)]
     public ICollection<PeerInfo>? Replicas { get; set; }
+
+    /// <summary>
+    /// Represents the duration of time since the cluster leader was elected.
+    /// </summary>
+    /// <remarks>Supported by server v2.12</remarks>
+    [System.Text.Json.Serialization.JsonPropertyName("leader_since")]
+    [System.Text.Json.Serialization.JsonIgnore(Condition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingDefault)]
+#if NET6_0
+    public DateTimeOffset LeaderSince { get; set; }
+#else
+    public DateTimeOffset LeaderSince { get; init; }
+#endif
+
+    /// <summary>
+    /// Indicates whether the account is a system account.
```

Comment:
whether the traffic account is the system account

https://github.com/scottf/jsm.go/blob/main/schema_source/jetstream/api/v1/definitions.json#L249